tests/file_storage/tests.py

```diff
@@ -306,6 +307,13 @@ def test_save_doesnt_close(self):
             self.assertFalse(file.closed)
             self.assertFalse(file.file.closed)
 
+    def test_file_save_with_pathlib_path(self):
+        p = Path('test.file')
+        self.assertFalse(self.storage.exists(p))
+        f = ContentFile('custom contents')
+        f_name = self.storage.save(p, f)
+        self.addCleanup(self.storage.delete, f_name)
```

Comment:
We can use `Path(f_name)` to check `storage.delete()` support for `pathlib.Path`.